src/components/Scenarios/FormFieldData.js

```diff
@@ -0,0 +1,24 @@
+import heroes from 'dotaconstants/build/heroes.json';
+import items from 'dotaconstants/build/items.json';
+import strings from '../../lang';
+
+export default function getFormFieldData(metadata) {
+  const { teamScenariosQueryParams, itemCost } = metadata;
+
+  return {
+    heroList: Object.keys(heroes).map(id => ({
```

Comment:
maybe provide a filter for game duration as well?

Reply:
game duration is not being tracked for that scenario actually. we could add it but it would increase the table by ~300k rows?

Reply:
or did you mean the lane role scenario? yeah I'll do that.